server/linter/linter.go

```diff
@@ -2,107 +2,81 @@ package linter
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/kubeshop/tracetest/server/linter/analyzer"
-	linter_plugin_common "github.com/kubeshop/tracetest/server/linter/plugins/common"
-	linter_plugin_security "github.com/kubeshop/tracetest/server/linter/plugins/security"
-	linter_plugin_standards "github.com/kubeshop/tracetest/server/linter/plugins/standards"
+	"github.com/kubeshop/tracetest/server/linter/plugins"
+	"github.com/kubeshop/tracetest/server/linter/rules"
 	"github.com/kubeshop/tracetest/server/model"
 )
 
 var (
-	AvailablePlugins = []model.Plugin{
-		linter_plugin_standards.NewStandardsPlugin(),
-		linter_plugin_security.NewSecurityPlugin(),
-		linter_plugin_common.NewCommonPlugin(),
-	}
+	commonPlugin = plugins.NewPlugin(
```

Comment:
Static registration of the linter configuration, we do not have much flexibility today, but in the future, the linter runner could generate a dynamic plugin registry with custom rules and plugins defined by user profile, test or test run levels